app/stream/index/chat/controller.js

```diff
@@ -82,7 +83,7 @@ export default Controller.extend({
 
     this.scrollToBottom(0); // scroll to bottom with 0 delay
 
-    // this.get('scroll').enable(this.$comments, this.onCommentsScroll);
+    this.get('scroll').enable(this.$comments, this.onCommentsScroll);
     this.$comments.on('scroll', run.bind(this, this.onCommentsScroll));
     this.$comments.on('touchmove', run.bind(this, this.onCommentsScroll));
 
```

Comment:
We don't need all of these - it's adding multiple scroll listeners. We should use enable and a callback to trigger onCommentsScroll, or some other neat pattern